### DISCUSSION TITLE
¿Que clases identificamos en el sistema biblioteca?

### DISCUSSION THREAD

**Discussion by nelsonvilte** [proyectos-tsdwad/integrador-modulo-fullstack#25] [Ideas]
## 👋 Welcome!
  📢Para ir colaborando con el desarrollo del Backend en el Sprint 1 podemos ir identificando 

  * Clases de la estructura del sistema, grupo de objetos con estructura y comportamiento similares.
  * Relaciones entre clases 💪.

  ✏️ Comenta a continuación tus ideas de clases y relaciones para el proyecto.

**Comment by nelsonvilte** [comment]
Las primeras clases que se me ocurren son:

- Clase **Libros** tiene los siguientes atributo:
  - Número de identificación único 
  - Titulo
  - Autor

- Clase **Autores** tiene los siguientes atributos
  - Numero de identificación único 
  - Nombre
  - Apellido
  - Género

**Reply by carlariossuarez** [reply]
Junto con @carlos-linero desarollamos ideas para las relaciones. Adjunto pdf
[UML Proyecto Biblioteca.pdf](https://github.com/proyectos-tsdwad/integrador-modulo-fullstack/files/9793373/UML.Proyecto.Biblioteca.pdf)

**Reply by nelsonvilte** [reply]
Este seria el diagrama para la base de datos que nos es util para el diagrama de clases
